service/src/main/java/com/ge/predix/acs/policy/evaluation/cache/DefaultPolicyEvaluationCacheCircuitBreaker.java

```diff
@@ -7,26 +7,24 @@
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Value;
-import org.springframework.stereotype.Component;
 
 import com.ge.predix.acs.privilege.management.dao.ResourceEntity;
 import com.ge.predix.acs.privilege.management.dao.SubjectEntity;
 import com.ge.predix.acs.rest.PolicyEvaluationResult;
 import com.netflix.hystrix.contrib.javanica.annotation.HystrixCommand;
 
-@Component
 public class DefaultPolicyEvaluationCacheCircuitBreaker implements PolicyEvaluationCacheCircuitBreaker {
     private static final Logger LOGGER = LoggerFactory.getLogger(DefaultPolicyEvaluationCacheCircuitBreaker.class);
 
     @Autowired
     private PolicyEvaluationCache cacheImpl;
-    @Value(value = "${DISABLE_CACHING:false}")
-    private boolean disableCaching;
+    @Value(value = "${ENABLE_CACHING:false}")
+    private boolean cachingEnabled;
```

Comment:
you could use PolicyEvaluationCacheConfig.isCachingEnabled (add this getter) to avoid setting default value in multiple places.